### PR TITLE
version 1 of the landsat 8 data example

### PR DESCRIPTION
**Pull request by Onyothi**
Created a Landsat 8 data example notebook. 
Jira ticket https://up42.atlassian.net/browse/UP-5273?atlOrigin=eyJpIjoiNzhhZGVhNjFkODI4NDAwZTkwMDdmYmM5Y2UyOWUzOWEiLCJwIjoiaiJ9 

Items:
* [ ] Ran test & live-tests
* [ ] Implemented (new) unit tests
* [ ] Removed credentials
* [ ] Removed argument pointing to staging
* [ ] Updated [documentation](sdk.up42.com)

For release:
* [ ] Bumped version
* [ ] Added changelog
* [ ] Updated announcement banner

### CODE REVIEW

File: examples/data-block-examples/landsat8-geotiff-example.ipynb [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "straight-candy",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/wilhelmina.nekoto/.pyenv/versions/UP42-py-dev/lib/python3.7/site-packages/pandas/compat/__init__.py:97: UserWarning: Could not import the lzma module. Your installed Python is incomplete. Attempting to use lzma compression will result in a RuntimeError.\n",
```
Comment (markusuwe) [inline examples/data-block-examples/landsat8-geotiff-example.ipynb:13]: Please either solve this problem or remove the output of the cell from the jupyter notebook if you think it is not important to resolve. In any case, should not be visible to users.

File: examples/data-block-examples/landsat8-geotiff-example.ipynb [hunk 2]
```diff
@@ -0,0 +1,326 @@
+{
```
Comment (markusuwe) [inline examples/data-block-examples/landsat8-geotiff-example.ipynb:1]: Please add an introductory text pointing out what the used blocks are and do. You can check other notebooks in the same directory for examples.

File: examples/data-block-examples/landsat8-geotiff-example.ipynb [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-03-15 11:51:31,084 - Authentication with UP42 successful!\n",
+      "2021-03-15 11:51:32,176 - Initialized Project(name: Landsat8-data-example, project_id: fbac95e6-1d78-4953-9485-b6fc072cb4c2, description: , createdAt: 2021-03-15T10:48:54.591911Z)\n"
+     ]
+    }
+   ],
+   "source": [
+    "up42.authenticate(project_id=\"fbac95e6-1d78-4953-9485-b6fc072cb4c2\", \n",
```
Comment (markusuwe) [inline examples/data-block-examples/landsat8-geotiff-example.ipynb]: Please remove those credentials. This is information that needs to kept secure under all circumstances and should never be put into a public facing github repository.
As this information is now already in git, the best solution is to delete this project permanently and use a new one for this activity. Then use the mechanisms you can see in other notebooks how to hide this information.

File: examples/data-block-examples/landsat8-geotiff-example.ipynb [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-03-15 12:01:53,155 - Getting existing workflows in project ...\n",
+      "2021-03-15 12:01:53,740 - Got 1 workflows for project fbac95e6-1d78-4953-9485-b6fc072cb4c2.\n",
+      "100%|██████████| 1/1 [00:00<00:00,  1.66it/s]\n",
+      "2021-03-15 12:01:54,344 - Using existing workflow: Landsat8-Geotiff-example - 2b4b6d62-2207-449c-a46d-658d5f960032\n",
+      "2021-03-15 12:01:58,264 - Added tasks to workflow: [{'name': 'sentinelhub-landsat8-aoiclipped:1', 'parentName': None, 'blockId': '95519b2d-09d7-4cd0-a321-4d6a46bef6c1'}]\n"
```
Comment (markusuwe) [inline examples/data-block-examples/landsat8-geotiff-example.ipynb:75]: Unfortunately you picked the wrong data block. I believe it should be this one, but best to confirm with @nb-42

**Review by markusuwe — CHANGES_REQUESTED** [review]
I believe you unfortunately picked the wrong data block, you took landsat-8 (geotiff), but the ticket says landsat-8 (geotiff-bundle). Please confirm with @nb-42 which block to use.

Other than that: this notebook will be customer facing which means it has to look great. Please:
- Add an introductory text
- Remove or update all comments only leaving those which help the user to understand what is going on
- Pick an image which is attractive, the one you used so far is full of clouds.

Most importantly though: please never put credentials (like the project id / key) into a public github repo.

File: examples/data-block-examples/landsat8-geotiff-example.ipynb (hunk 3 above)
Comment (Onyothi) [reply examples/data-block-examples/landsat8-geotiff-example.ipynb]: I will proceed to delete the project and create a new one.

File: examples/data-block-examples/landsat8-geotiff-example.ipynb (hunk 2 above)
Comment (Onyothi) [reply examples/data-block-examples/landsat8-geotiff-example.ipynb:1]: Just added the description, however I will update it the new notebook.

**Review by thaisbendixen — APPROVED** [review]
Nice!! :)

**Review by markusuwe — APPROVED** [review]
LGTM
Nice, so much better now. Looking forward to the next ones. :)

**Review by nekhtiari — APPROVED** [review]
:heart:

**Review by marooned20 — APPROVED** [review]
Awesome! Great job!

**Pull request merged**